ergo-p2p/Cargo.toml

```diff
@@ -14,14 +14,18 @@ exclude = [
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-
 sigma-ser = { version = "^0.3.0", path = "../sigma-ser" }
 sigma-util = { version = "^0.3.0", path = "../sigma-util" }
 thiserror = "1"
 derive_more = "0.99"
 bounded-vec = { version = "^0.5.0" }
 proptest-derive = {version = "0.3.0", optional = true }
 
+futures = "0.3"
```

Comment:
Super early in the draft but just wanted to mention:

`tokio` might be a good choice for the networking/async layer if you haven't looked at it 😃

Reply:
Oh, there will be `tokio` in it for sure. :) Thanks for checking in!